src/Sobol.jl

```diff
@@ -1,8 +1,7 @@
-VERSION < v"0.7.0-beta2.199" && __precompile__()
+__precompile__()
```

Comment:
This change doesn't look right — `__precompile__()` is now the default, so it doesn't need to be called on 0.7 or newer.